### ISSUE TITLE
Datagrid: checkbox state inside a cell is not announced.

### ISSUE THREAD

**Issue by JBismonte** [infor-design/enterprise#4790]
**Describe the bug**
[VPAT] if there is a checkbox inside a cell inside a soho-datagrid, the checkbox's state is not announced to the assistive technology. because the focus is in the outer cell instead of the  element with role="checkbox" and an aria-checked attribute inside of it.

**To Reproduce**

1. Use a screen reader (Chrome Vox/NVDA etc.).
2. Go to https://master-enterprise.demo.design.infor.com/components/datagrid/test-editable-checkboxes.html
3. Navigate to a cell with checkbox, the checkbox is not announced on the screen reader and also if the checkbox is checked/not checked is not announced also, probably because of the focus is at the outer cell instead of the checkbox.

**Expected behavior**
Whan navigating to a cell with a checkbox, The focus should be on the element with the aria checked attribute.

**Version**

- ids-enterprise: 4.36.2

**Screenshots**
![image](https://user-images.githubusercontent.com/29673378/106288872-86fe9080-6283-11eb-8931-cbd51052cc18.png)

**Platform**
 - Infor Application/Team Name: Landmark Client
 - Device: (if applicable) : N\A
 - OS Version: Windows 10
 - Browser Name: Chrome
 - Browser Version: 88.0.4324.104 (Official Build) (64-bit)

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-33868

**Comment by janahintal** [comment]
This issue is now resolved. Thank you. 
![image](https://user-images.githubusercontent.com/45757016/109461330-f04d0b80-7a9c-11eb-92a4-6d493c941c11.png)

![image](https://user-images.githubusercontent.com/45757016/109461342-f5aa5600-7a9c-11eb-8b32-b2166a339509.png)
